package.json

```diff
@@ -12,8 +12,8 @@
     "boom": "7.x.x",
     "haute-couture": "3.x.x",
     "joi": "13.x.x",
-    "objection": "0.9.x",
-    "schwifty": "3.x.x"
+    "objection": "1.0.x",
```

Comment:
I think this oughtta be `1.x.x`.  Before v1 breaking changes go into minor-version releases, but afterwards they must go into major-version releases (under semver).